jsx/Form.js

```diff
@@ -132,20 +109,53 @@ var FormElement = React.createClass({
       </form>
     );
   }
-});
+}
+
+FormElement.propTypes = {
+  name: PropTypes.string.isRequired,
```

Comment:
should be React.PropTypes. adding this might fix the need to manually import above